packages/slice-machine/start/index.js

```diff
@@ -77,11 +77,8 @@ function start({ cwd, port }, callback) {
   });
 
   smServer.stdout.on("data", function (data) {
-    const lns = data.toString().split("=");
+    const lns = data.toString().split("Server running");
```

Comment:
The fix that you made by reading the function SMConfig is good, so why did you change this split ?
What does it change ?

Reply:
This line of code is here to replace "server running : localhost...." by the green info box that we know

Reply:
Before the dev server write "p=9999", now the server write "Server Running: localhost:9999"

Reply:
This line of code is here to replace "server running" by the infobox, the regression was introduced when we change the Express message in the setup drawer PR